sql-db/hibernate-reactive/src/main/java/io/quarkus/ts/reactive/PanacheEndpoint.java

```diff
@@ -0,0 +1,135 @@
+package io.quarkus.ts.reactive;
+
+import javax.inject.Inject;
+import javax.ws.rs.Consumes;
+import javax.ws.rs.DELETE;
+import javax.ws.rs.GET;
+import javax.ws.rs.POST;
+import javax.ws.rs.PUT;
+import javax.ws.rs.Path;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+
+import io.smallrye.mutiny.Multi;
+import io.smallrye.mutiny.Uni;
+
+@Path("/library")
+@Produces(MediaType.APPLICATION_JSON)
+@Consumes(MediaType.APPLICATION_JSON)
+public class PanacheEndpoint {
+
+    @Inject
+    AuthorRepository authors;
+
+    @GET
+    @Path("books")
+    public Multi<String> all() {
+        return Book.all()
+                .map(Book::getTitle);
+    }
+
+    @GET
+    @Path("books/{id}")
+    public Uni<Response> find(Integer id) {
+        return Book.byId(id)
+                .map(book -> book == null
+                        ? Response.status(Response.Status.NOT_FOUND)
+                        : Response.ok(book.getTitle()))
+                .map(Response.ResponseBuilder::build);
+    }
+
+    @GET
+    @Path("isbn/{id}")
+    public Uni<Response> findISBN(Integer id) {
+        return Book.byId(id)
+                .map(book -> book == null
+                        ? Response.status(Response.Status.NOT_FOUND)
+                        : Response.ok(book.getISBN()))
+                .map(Response.ResponseBuilder::build);
+    }
+
+    @PUT
+    @Path("isbn/{id}/{isbn}")
+    public Uni<Response> changeISBN(Integer id, Long isbn) {
+        return Book.byId(id)
+                .flatMap(book -> {
+                    book.setISBN(isbn);
+                    return book.persistAndFlush();
+                })
+                .map(book -> book == null
+                        ? Response.status(Response.Status.NOT_FOUND)
+                        : Response.ok())
+                .map(Response.ResponseBuilder::build);
+    }
+
+    @PUT
+    @Path("books/{author}/{name}")
+    public Uni<Response> createBook(Integer author, String name) {
+        return Book.create(author, name)
+                .map(nothing -> Response.status(Response.Status.CREATED))
+                .onFailure().recoverWithItem(error -> Response.status(Response.Status.BAD_REQUEST).entity(error.getMessage()))
+                .map(Response.ResponseBuilder::build);
+    }
+
+    @GET
+    @Path("books/author/{name}")
+    public Multi<String> search(String name) {
+        return authors.findByName(name)
+                .flatMap(Author::getBooks)
+                .map(Book::getTitle);
+    }
+
+    @GET
+    @Path("author")
+    // TODO: Return Json, when this will be fixed: https://github.com/quarkusio/quarkus/issues/18043
+    public Multi<String> author() {
```

Comment:
```suggestion
    public Multi<String> getAllAuthors() {
```
please go through method names and adjust their names

Reply:
Method renamed